components/drop-zone/provider.js

```diff
@@ -133,11 +132,15 @@ class DropZoneProvider extends Component {
 				isDraggingOverDocument,
 			} );
 		} );
+
 		this.setState( {
 			isDraggingOverDocument,
 			hoveredDropZone,
 			position,
 		} );
+
+		event.preventDefault();
+		event.stopPropagation();
```

Comment:
Reminder to check again and probably remove this last line. No need to stop propagation here.

Reply:
Any decision here?